Documentation/connectors/github.md

```diff
@@ -67,6 +67,9 @@ connectors:
     #   - ['acme:site-reliability-engineers'] for 'slug'
     #   - ['acme:Site Reliability Engineers', 'acme:site-reliability-engineers'] for 'both'
     teamNameField: slug
+    # flag which will switch from using the internal GitHub id to the users handle (@mention) as the user id.
+    # It is possible for a user to change their own user name but it is very rare for them to do so
+    useLoginAsId: false
```

Comment:
[nit] let's make id `Id`? What do you think?
```suggestion
    useLoginAsID: false
```

Reply:
I normally prefer `Id` but will change as it is not my codebase (there are 135 occurences of `UserID` and 32 of `UserId`)  :)